### ISSUE TITLE
SB3 Dedicated Server Crashing

### ISSUE THREAD

**Issue by trevorgolden** [FTBTeam/FTB-Modpack-Issues#2320]
### Modpack

FTB StoneBlock 3

### Modpack version

1.4.3

### Log Files

Server Log running Magma: https://paste.feed-the-beast.com/view/dfa131ac

Server Log running Arclight: https://paste.feed-the-beast.com/view/265090c4

Crash Log Running Magma: https://paste.feed-the-beast.com/view/7bf147c4

Crash Log Running Arclight: https://paste.feed-the-beast.com/view/b74e7b37

### Describe the bug

Other stoneblocks are running perfectly fine thank god, however, upon teleporting to my stoneblock, the server crashes, I have had this server online for a little over a month now updating as the updates become available of course, and have just started having issues. I can go to other stoneblocks just fine 

Accidentally posted this report in the og stoneblock github earlier so I will add the to add part here as well

server was running fine night before when I logged off, when I logged back in, It crashed immediately, teleported myself to the server spawn after server started but before logging in, and server works as long as my stoneblock is not populated, there are 24 force loaded chunks in my stoneblock, all of which have been in place for well over 2 weeks now with no issues, which suggests to me that those chunks are not the issue.

### Steps to reproduce

Upon teleporting to my stoneblock on the server, the server crashes

1.Spawn/login to server
2./ftbstoneblock home
3.Crash

### Expected behaviour

Expect to be able to issue commands, break blocks, and continue gameplay upon using the /ftbstoneblock home command

### Additional information

Modpack Version: 1.4.3
Dedicated Server running on i9-9900K with 12G Ram allocated

EDIT: stated wrong pack version

**Comment by Raidobw2** [comment]
Sorry for replying so late, it seems both the crashes on your servers point to different problems, kinda, so it's odd.

The one on Magma is crashing on an issue with the lighting engine, which is odd.

The one on Arclight is crashing because of some entity code that's affecting an Enderman. Arclight's mixins are being mentioned: `mixins.arclight.core.json`, which could be causing the issue.

It's possible Magma or Arclight are both causing issues, depending on what they have installed... Which is impossible for me to guess.

In short, does this happen on a standard Forge for server installation, without added mods/plugins?
